optuna/multi_objective/samplers/_nsga2.py

```diff
@@ -259,11 +260,12 @@ def _select_elite_population(
     def _select_parent(
         self,
         study: "multi_objective.study.MultiObjectiveStudy",
-        population: List["multi_objective.trial.FrozenMultiObjectiveTrial"],
+        population: Sequence["multi_objective.trial.FrozenMultiObjectiveTrial"],
     ) -> "multi_objective.trial.FrozenMultiObjectiveTrial":
         # TODO(ohta): Consider to allow users to specify the number of parent candidates.
-        candidate0 = self._rng.choice(population)
-        candidate1 = self._rng.choice(population)
+        population_size = len(population)
+        candidate0 = population[self._rng.choice(population_size)]
+        candidate1 = population[self._rng.choice(population_size)]
```

Comment:
These to `choice` calls could probably be unified to a snigle `self._rng.choice(population_size, size=2, replace=True)` but I left it outside this type hint fix.